CovertInfiltration/Config/XComGameData.ini

```diff
@@ -142,12 +148,6 @@ NumDarkEventsThirdMonth = 3
 +SITREPS_EXCLUSIVE_BUCKETS=(SitReps[0]="ShoddyIntel", SitReps[1]="WellRehearsed")
 +SITREPS_EXCLUSIVE_BUCKETS=(SitReps[0]="Phalanx_CI", SitReps[1]="Congregation")
 
-+SITREPS_MISSION_BLACKLIST=(MissionType="Terror", SitRep="LightningStrike")
-+SITREPS_MISSION_BLACKLIST=(MissionType="Terror", SitRep="AdventAirPatrols")
-+SITREPS_MISSION_BLACKLIST=(MissionType="Terror", SitRep="IntelligenceLeak")
-+SITREPS_MISSION_BLACKLIST=(MissionType="Terror", SitRep="ShoddyIntel")
-+SITREPS_MISSION_BLACKLIST=(MissionType="Terror", SitRep="WellRehearsed")
-+SITREPS_MISSION_BLACKLIST=(MissionType="Terror", SitRep="GunneryEmplacements")
```

Comment:
🤔

Reply:
We no longer need these since retaliations won't be getting random pos/neg sitreps anymore